client/src/App.js

```diff
@@ -1,10 +1,11 @@
 import React, { useState, useEffect } from 'react'
+import { Link } from "react-router-dom";
 
 function App() {
   const [data, setData] = useState({})
 
   useEffect(() => {
-    fetch("/endpoint").then(
+    fetch("http://localhost:3000/endpoint").then(
```

Comment:
Was this the only way to fetch the router by adding the http stuff instead of adding just the endpoint?  We would change the endpoint to login so it starts there and after login redirect to homepage. When we get everything working ofc

Reply:
The /endpoint alone was causing a Proxy Error issue, even though the react app was still running. This was the one way that I found to get rid of the proxy error.